packages/opentelemetry-exporter-jaeger/src/jaeger.ts

```diff
@@ -35,7 +35,7 @@ export class JaegerExporter implements SpanExporter {
 
   constructor(config: jaegerTypes.ExporterConfig) {
     const localConfig = Object.assign({}, config);
-    this._logger = localConfig.logger || new api.NoopLogger();
+    this._diagLogger = getDiagLoggerFromConfig(localConfig);
```

Comment:
This will fallback to the api.diag rather than the previous NoopLogger, so while the default api.diag is a Noop Diag Logger, if it is set this would be a change in functionality.